test-suite/shared/src/test/scala/org/scalajs/testsuite/niobuffer/BufferFactory.scala

```diff
@@ -55,8 +59,7 @@ object BufferFactory {
 
     implicit def elemFromInt(value: Int): ElementType = value.toByte
 
-    implicit def elemToJSAny(elem: ElementType): js.Any =
-      js.Any.fromByte(elem)
+    implicit def elemToAnyRef(elem: ElementType): AnyRef = Byte.box(elem)
```

Comment:
Just `elem` should implicitly convert to `AnyRef` through `Predef.byte2Byte`, shouldn't it?